static/src/js/components/AdminPreferencesMetrics/AdminPreferencesMetricsList.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { startCase } from 'lodash-es'
+
+import { Table } from 'react-bootstrap'
+
+import Spinner from '../Spinner/Spinner'
+
+import './AdminPreferencesMetricsList.scss'
+
+/**
+ * Creates the tables for all the preferences counts
+ * @param {Object} preferences Object with the preferences with an array of the field value pairs
+ * @returns preference tables
+ */
+const createPreferencesTable = (preferences) => {
+  const tables = Object.keys(preferences).map((key) => {
+    const header = (
+      <thead key={`${key}_header`}>
+        <tr className="admin-preferences-metrics-list__table-row" key={`${preferences[key]}_header`}>
+          {
+            // Pulling out the values (not the count)
+            Object.values(preferences[key]).map(([field, count]) => (
+              <th aria-label={`${field} ${count} Header`} key={`${field}}`}>{field}</th>
```

Comment:
I don't think you need this aria label here. The `th` element can be assumed to be a header. If it's for testing, I think table headers are automatically assigned an aria role that you should be able to select.

Reply:
This was specifically brought up by Matthew. He wanted to use an aria-label to more accurately pinpoint the field-values pairs we're checking. @macrouch